src/svc/annotationd/paths/annotations.yml

```diff
@@ -3,6 +3,11 @@ post:
     - Annotations
   operationId: createAnnotations
   summary: Create annotations
+  parameters:
+    - name: orgID
+      in: query
+      schema:
+        type: string
```

Comment:
Since orgID as a query parameter is oss-only (annotations in cloud derives the org-id from the token), consider creating a `src/svc/annotationd-oss` that defines these custom paths and re-uses whatever it needs in terms of schemas.